catalyst-toolbox-cli/src/cli/notifications/send.rs

```diff
@@ -0,0 +1,130 @@
+use crate::cli::notifications::{
+    api_params::ApiParams,
+    requests::create_message::{CreateMessage, DATETIME_FMT},
+    responses::create_message::CreateMessageResponse,
+    Error,
+};
+
+use chrono::{DateTime, FixedOffset};
+use reqwest::{blocking::Client, StatusCode, Url};
+use structopt::StructOpt;
+
+use std::io::Read;
+use std::path::PathBuf;
+
+use crate::cli::notifications::requests::create_message::{
```

Comment:
Move this to the top if we are grouping `crate` imports together.

Reply:
Damn you autoimport!

Reply:
More like "damn you `rustfmt`", IIRC imports ordering is still in nightly.